code/modules/vending/medical.dm

```diff
@@ -10,6 +10,8 @@
 					/obj/item/reagent_containers/dropper = 3,
 					/obj/item/healthanalyzer = 4,
 					/obj/item/wrench/medical = 1,
+					/obj/item/extraction_pack = 10,
```

Comment:
if you want them to cost money for medical staff then make them premium items

Reply:
Fikou it's okay, please stop crying.

Reply:
use the review functionality then noob